vcpkg/libtorrent/CONTROL

```diff
@@ -0,0 +1,26 @@
+Source: libtorrent
```

Comment:
Should we add a readme.md in the same folder and put a link to this file origin in it?
This is assuming we are interested in following/merging vcpkg changes.

Reply:
>we are interested in following/merging vcpkg changes.

I think Yes, more or less.
>Should we add a readme.md in the same folder and put a link to this file origin in it?

I think it's not that hard to find the appropriate port in vcpkg repository. However, I don't mind adding "readme" if you specify in detail what exactly it should contain (I'm not too good at this).

Reply:
>I think it's not that hard to find the appropriate port in vcpkg repository. However, I don't mind adding "readme" if you specify in detail what exactly it should contain (I'm not too good at this).

We can put something easy as a start:
```
Files are taken from upstream repository: https://github.com/microsoft/vcpkg/tree/master/ports/libtorrent
```

Another question: is our self-maintained vcpkg port recommended for general use or it's only for CI? If it is the latter then we should mention it in readme: `This port is for satisfying CI needs, NOT recommended for general use.`

Reply:
>We can put something easy as a start:

Done.
> is our self-maintained vcpkg port recommended for general use or it's only for CI?

Only for CI. Unless someone want to maintain it for general use (but in this case it can be maintained upstream, isn't it?).
>we should mention it in readme

Done.
Should I also move `vcpkg` folder under .`github/workflows`?

Reply:
>Should I also move vcpkg folder under .github/workflows?

Not sure, maybe in the future other CI would use it.

Reply:
>maybe in the future other CI would use it.

👍

Reply:
PR updated to fix "file health" issue.

Reply:
>maybe in the future other CI would use it.

So what do you still think about Appveyor? Now we can't go any further because of it, and it will be so every time we want to upgrade the version of libtorrent (or some other library).

Reply:
>So what do you still think about Appveyor? Now we can't go any further because of it, and it will be so every time we want to upgrade the version of libtorrent (or some other library).

There are 2 ways I can think of:
The first one is disable it and we loose testing qmake on MSVC.
The second one is we open another repo to hold the appveyor dependencies (this file: https://builds.shiki.hu/appveyor/qbt_libraries.7z) and we submit updates when needed (in binary form). This way we move the burden off @sledgehammer999.

Reply:
Or we can add a build with qmake on GitHub Actions. I tried to deal with this recently, but really I don't want to continue, as it takes time away from other tasks I'm currently doing.